tools/metrics/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+	"time"
+)
+
+func regressions() (string, error) {
+	buf := bytes.NewBufferString("")
+	start := time.Now()
+	weeks := 7
+	srr, err := QueryRegressions(NewDefaultQueryClient(), time.Now(), Weekly, weeks)
+	elapsed := time.Since(start)
+	fmt.Fprintf(buf, "Query took %v\n", elapsed)
+	if err != nil {
+		return "", err
```

Comment:
in case of error you return empty string, hence the lines 16-17 have no effect here. please add error handling directly after `QueryRegressions`.

Reply:
Done